.circleci/ci/.nvmrc

```diff
@@ -1 +1 @@
-16.10.0
\ No newline at end of file
+20
```

Comment:
I have no opinion for `20` or `20.x.y`, but maybe @Okhelifi and @ThibaudAV have one?

Reply:
I don't have a strong opinion on it 🤷🏼

Reply:
I don't think there's too much risk and it keeps us up to date. If there's a problem, we can fix the minor later 🤔